### ISSUE TITLE
Limited shoulder abduction

### ISSUE THREAD

**Issue by JillMiller** [obophenotype/human-phenotype-ontology#5904]
**Preferred term label:** Limited shoulder abduction

**Synonyms**

**Definition (free text, please give PubMed ID)** Reduced range of motion of the shoulder resulting in difficulty moving the arm away from the midline of the body., PMID:32296967

**Parent term (use hpo.jax.org/app)** Limited shoulder movement HP:0006467

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
added
